tests/FakeItEasy.Specs/ConfiguringPropertySetterSpecs.cs

```diff
@@ -428,6 +432,102 @@ public interface IHaveInterestingProperties
                 .x(() => wasConfiguredBehaviorUsed.Should().Be(fateOfConfiguredBehavior == "used"));
         }
 
+        [Scenario]
+        public static void ConfiguringSetterFromBaseInterfaceOnFakedInterface(
+            IHaveInterestingDerivedProperties subject,
+            bool wasConfiguredBehaviorUsed)
+        {
+            "Given an interface with a read/write property"
+                .See<IHaveInterestingProperties>();
+
+            "And an interface that implements it"

```

Comment:
An interface doesn't implement anything :wink: ([yet!](https://github.com/dotnet/csharplang/issues/52)). It just inherits from another interface.

Reply:
Good point. Will push a fixup.